src/main/java/io/openliberty/tools/intellij/util/DebugModeHandler.java

```diff
@@ -161,9 +165,9 @@ private void createAndRunDebugConfiguration(ProgressIndicator indicator, Liberty
      */
     private String waitForSocketActivation(ProgressIndicator monitor, LibertyModule libertyModule, String host, int debugPort) throws Exception {
         byte[] handshakeString = "JDWP-Handshake".getBytes(StandardCharsets.US_ASCII);
-        int retryLimit = 300;
-        int envReadMinLimit = 45;
-        int envReadInterval = 5;
+        int retryLimit = 120; // wait up to 60s for dev mode to start (polling rate 2 per second, 500ms)
```

Comment:
Since attachment seems to be an asynchronous process, we should consider wait times longer than roughly 1 minute. Slow systems along with Gradle built projects with many dependencies may prematurely report that the debugger could not be attached. Increasing the time to wait (i.e. roughly 2/3 minutes) maybe more sensible.